docs/elements/jira/tips.md

```diff
@@ -0,0 +1,20 @@
+---
+heading: Jira
+seo: Tips | Jira | Cloud Elements API Docs
+title: Tips
+description: Jira Tips
+layout: sidebarelementdoc
+breadcrumbs: /docs/elements.html
+elementId: 1
+parent: Back to Element Guides
+order: 75
+---
+
+# Tips
+
+The Tips section includes more information about the element including FAQs, endpoint limitations, and general tips.
+
+
+## General
```

Comment:
Events...
although if there is only, we probably don't need to separate out in headings.